realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -79,6 +79,19 @@
  * A standard pattern for working with Realm in Android activities can be seen below:
  * <p>
  * <pre>
+ * public class RealmApplication extends Application {
+ *
+ *     private static RealmConfiguration defaultRealmConfig;
```

Comment:
The Application class is only created when starting the app. The only case you need to reference the configuration (instead of using `getDefaultInstance`), is if you want to manually delete or migrate the Realm, but for the Javadoc I think we can go with the minimal setup. I would remove the static var.